### PR TITLE
Provide more deferred deletion skipping context

### PR DESCRIPTION
**Pull request by cevich**

### CODE REVIEW

**Comment by cevich** [comment]
N/B: Untested/first-draft...letting CI take care of that.

File: subtests/docker_cli/deferred_deletion/deferred_deletion.py [hunk 1]
```diff
@@ -48,15 +48,15 @@ def initialize(self):
...
+                       DockerTestNAError)
 
         docker_cmdline = dockertest.docker_daemon.cmdline()
-        if "dm.use_deferred_deletion=true" not in str(docker_cmdline):
-            raise DockerTestNAError("deferred deletion not enabled in"
-                                    " docker daemon command-line options.")
+        self.failif_not_in("dm.use_deferred_deletion=true", str(docker_cmdline),
+                           "docker daemon command-line options",
+                           DockerTestNAError)
+
```
Comment (edsantiago) [inline subtests/docker_cli/deferred_deletion/deferred_deletion.py]: I don't know why github isn't showing it, but this has trailing whitespace. You may want to look at your local commit hooks and make sure that `pre-commit` is enabled.

Comment (cevich) [inline subtests/docker_cli/deferred_deletion/deferred_deletion.py]: ack..they're all messed up.

**Comment by cevich** [comment]
Okay, part way there.  Debug log is much more helpful now:

```
DockerTestNAError: Expected string 'dm.use_deferred_deletion=true' not in docker daemon 
command-line options '['/usr/bin/dockerd-latest', '--add-runtime', 'docker-runc=/usr/libexec/docker/docker-
runc-latest', '--default-runtime=docker-runc', '--authorization-plugin=rhel-push-plugin', '--exec-opt', 
'native.cgroupdriver=systemd', '--userland-proxy-path=/usr/libexec/docker/docker-proxy-latest', '-g', 
'/var/lib/docker-latest', '--selinux-enabled', '--log-driver=journald', '--signature-verification=false', '--storage-
driver', 'devicemapper', '--storage-opt', 'dm.fs=xfs', '--storage-opt', 'dm.thinpooldev=/dev/mapper
/docker_vg-docker--latest--pool', '--storage-opt', 'dm.use_deferred_removal=true', '--add-registry', 
'registry.access.redhat.com', '']'
```

File: subtests/docker_cli/deferred_deletion/deferred_deletion.py (hunk 1 above)
Comment (cevich) [inline subtests/docker_cli/deferred_deletion/deferred_deletion.py]: okay, fixed and fixed my ``vim`` too.

**Comment by cevich** [comment]
hellloooooo github....I pushed my changes....WTF are they!?!?!?

**Comment by cevich** [comment]
uggg where's my commit!  Should be d9401ad16

**Comment by cevich** [comment]
Ugg, I give up...re-rolling a new PR.

**Pull request closed without merging**